services/gitlab/gitlab-merge-requests.spec.js

```diff
@@ -1,17 +1,19 @@
 import { test, given } from 'sazerac'
+import nock from 'nock'
+import { expect } from 'chai'
+import { cleanUpNockAfterEach, defaultContext } from '../test-helpers.js'
 import GitlabMergeRequests from './gitlab-merge-requests.service.js'
+import GitLabTag from './gitlab-tag.service.js'
```

Comment:
We shouldn't need/be using this here because this file contains unit tests for the MergeRequests class

Reply:
ops